01week/rockPaperScissors.js

```diff
@@ -7,11 +7,18 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
-
+// Compare hand1 and hand2 inputs and spit out a result
+// written inside a whole function using if statements
+// rock beats scissors; paper beats rock; scissors beat paper
+// put the bulk in argument to minimize lines using pipes and ampersands
 function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+  if (hand1 === hand2){
+    return 'Tie!';
+  } else if (hand1 === 'rock' && hand2 === 'scissors' || hand1 === 'paper' && hand2 === 'rock' || hand1 === 'scissors' && hand2 === 'paper'){
```

Comment:
You need to wrap each scenario in parentheses. Remember order of operations and PEDMAS. I.E `else if ( (hand1 === 'rock' && hand2 === 'scissors') || (hand1 === 'paper' && hand2 === 'rock') || .......`

Reply:
I wrote another else if swapping out hand1 and hand 2 and replace the else with 'Someone is trying to cheat'

Also i wrapped the cases in parenthesis! Pull request to follow....